### PR TITLE
DDCE-5180 - Play 3.0 upgrade

### PR DESCRIPTION
**Pull request by frikit**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -3,12 +3,19 @@ import uk.gov.hmrc.DefaultBuildSettings
...
+)
+
 lazy val scoverageSettings = {
   import scoverage.ScoverageKeys
   Seq(
-    ScoverageKeys.coverageExcludedPackages := "<empty>;.*Reverse.*;.*.Routes.*;prod.*;testOnlyDoNotUseInProd.*;testOnlyDoNotUseInAppConf.*;" +
-      ".*BuildInfo.*;app.*;prod.*;config.*",
-    ScoverageKeys.coverageMinimumStmtTotal := 95,
+    ScoverageKeys.coverageExcludedFiles := excludedPackages.mkString(";"),
+    ScoverageKeys.coverageMinimumStmtTotal := 97,
```
Comment (MRBeresford) [inline build.sbt:18]: Is there a reason this was increased?

**Review by MRBeresford — COMMENTED** [review]
One really minor question, just running the ATs now if they pass will approve 👍

File: build.sbt (hunk 1 above)
Comment (frikit) [reply build.sbt:18]: it is reflecting real number now, maybe it increased/decreased after upgrade but this is real value now

**Review by MRBeresford — APPROVED** [review]
TrustStore ATs passed 👍

**Pull request merged**